invokeai/frontend/web/src/features/modelManagerV2/subpanels/ModelManagerPanel/ModelListItem.tsx

```diff
@@ -108,6 +108,9 @@ const ModelListItem = ({ model }: ModelListItemProps) => {
           <ModelBaseBadge base={model.base} />
           <ModelFormatBadge format={model.format} />
         </Flex>
+        <Text>
+          {Intl.NumberFormat(undefined, { maximumFractionDigits: 2 }).format(model.size / (1024 ** 3))} GB
```

Comment:
This feels like an unfortunately verbose way to spell `{model.size:.2f}`, is there a more concise way to do it?

Also, do we need to localize `GB`?